modular_skyrat/modules/modular_reagents/code/reagents/medicine.dm

```diff
@@ -0,0 +1,47 @@
+/datum/reagent/medicine/lidocaine
+	name = "Lidocaine"
+	description = "A numbing agent used often for surgeries, metabolizes slowly."
+	reagent_state = LIQUID
+	color = "#6dbdbd" // 109, 189, 189
+	metabolization_rate = 0.2 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+	ph = 6.09
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	addiction_types = list(/datum/addiction/opiods = 20)
+	inverse_chem_val = 0.55
+	inverse_chem = /datum/reagent/inverse/lidocaine
+
+/datum/reagent/medicine/lidocaine/on_mob_metabolize(mob/living/L)
+	..()
+	ADD_TRAIT(L, TRAIT_NUMBED, src)
+	L.throw_alert("numbed", /atom/movable/screen/alert/numbed)
+
+/datum/reagent/medicine/lidocaine/on_mob_end_metabolize(mob/living/L)
+	REMOVE_TRAIT(L, TRAIT_NUMBED, src)
+	L.clear_alert("numbed")
+	..()
+
+/datum/reagent/medicine/lidocaine/overdose_process(mob/living/M, delta_time, times_fired)
+	var/obj/item/organ/heart/our_heart = M.getorganslot(ORGAN_SLOT_HEART)
+	our_heart.applyOrganDamage(3 * REM * delta_time, 0)
+	..()
```

Comment:
The second argument of applyOrganDamage is essentially a clamp that keeps you from dealing more than X total damage. This will never actually deal heart damage as you've got it set to 0. Additionally it's more common to use adjustOrganLoss instead of applyOrganDamage for chems, as it does a check to see if the target has some sort of `GODMODE` first.

```suggestion
/datum/reagent/medicine/lidocaine/overdose_process(mob/living/M, delta_time, times_fired)
	M.adjustOrganLoss(ORGAN_SLOT_HEART,3 * REM * delta_time)
	..()
```
Also as it is now, this is _extremely_ lethal if overdosed. 23u will kill someone. You could also do something like limiting the amount of heart damage it does by going `M.adjustOrganLoss(ORGAN_SLOT_HEART,3 * REM * delta_time, 75)` or something, in this case it'd cap out at 75 heart damage so it'd never actually kill a person from heart failure.

Reply:
Resolved with new commit